kuksa_feeders/gps2val/config/gpsd_feeder.ini

```diff
@@ -1,4 +1,5 @@
 [kuksa_val]
+ip = 192.168.0.183
```

Comment:
Config detail for a specific setup? Should this be here? 127.0.0.1 seems more useful?